src/main/java/liquibase/ext/opensearch/database/OpenSearchLiquibaseDatabase.java

```diff
@@ -10,6 +10,7 @@ public class OpenSearchLiquibaseDatabase extends AbstractNoSqlDatabase {
     public static final String PRODUCT_NAME = "OpenSearch";
     public static final String PRODUCT_SHORT_NAME = "opensearch";
     public static final String OPENSEARCH_PREFIX = PRODUCT_SHORT_NAME + ":";
+    public static final String OPENSEARCH_URI_SEPARATOR = ";";
```

Comment:
I would expect `,` as a separator to be fair, it is the way multiple hosts are configured at the moment:

```suggestion
    public static final String OPENSEARCH_URI_SEPARATOR = ",";
```

So it would look like; `opensearch:host1:9200,host2:9200,host3:9200`

Reply:
good catch, thanks! ✔️